src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -1790,21 +1808,21 @@ internal async Task<bool> ReconfigureAsync(bool first, bool reconfigureAll, LogP
                                 var aex = task.Exception;
                                 foreach (var ex in aex.InnerExceptions)
                                 {
-                                    log?.WriteLine($"{Format.ToString(server)}: Faulted: {ex.Message}");
+                                    log?.WriteLine($"  {Format.ToString(server)}: Faulted: {ex.Message}");
```

Comment:
Are these `Format.ToString()` calls necessary? It looks like `ServerEndPoint` already overrides ToString to do that

Reply:
Good question - I would like to change these _everywhere_ but going consistent for this PR and want to do those all at once :)